com.unity.template.renderstreaming-hd/Assets/Scripts/Signaling/WebSocketSignaling.cs

```diff
@@ -0,0 +1,218 @@
+﻿using System;
+using System.Security.Authentication;
+using System.Text;
+using System.Threading;
+using Unity.WebRTC;
+using UnityEngine;
+using WebSocketSharp;
+
+namespace Unity.RenderStreaming.Signaling
+{
+    public class WebSocketSignaling : ISignaling
```

Comment:
It seems that there are a lot of common code between FurioosSignaling and WebSocketSignaling.
Can we put these common codes in one place ?
For example, by making FurioosSignaling derive from WebSocketSignaling.

Reply:
It's true that we're using similar code, but it's hard to unify them because they're slightly different.
Different : Initialize process , Messaging class

Reply:
Ok